### PR TITLE
Add link reset placeholder

### PR DESCRIPTION
**Pull request by kmaric69**
I've added a placeholder for resetting the default browser style on anchor links.

### CODE REVIEW

File: styles/scss/placeholders/_link.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+/// ```
+///
+/// @access public
+/// @author Kristijan Maric
+/// @example
+/// .test {
+///   @extend %link-reset;
+/// }
+
+%list-reset {
```
Comment (IvanGrginovInf) [inline styles/scss/placeholders/_link.scss:23]:
```suggestion
%link-reset {
```

Comment (IvanGrginovInf) [reply styles/scss/placeholders/_link.scss:23]: Also I believe the filename should be the same as the placeholder (so `link-reset.scss`).

Comment (kmaric69) [reply styles/scss/placeholders/_link.scss]: Not a bad idea, should I change it for all files that are `-reset`?

**Review by gabriel-glo — APPROVED** [review]

**Review by dingo-d — APPROVED** [review]

**Review by volfkarlo — APPROVED** [review]

**Comment by dingo-d** [comment]
@kmaric69 Feel free to merge this 😉

**Review by IvanGrginovInf — APPROVED** [review]

**Pull request merged**